### DISCUSSION TITLE
Coming back to GitHub after a long time.

### DISCUSSION THREAD

**Discussion by jack0194** [community/community#68745] [New to GitHub] [answered]
Hey everyone,
So I've decided to jump back on to Github. It's been a while since I've been on there though. I'm just trying to see whether I'm going the best way about this? Is the best thing to do, to go into the repository and work on existing projects and project issues? Or Do I find a project template and do that first?

Thanks, everyone.

**Comment by vishalkrishna8** [comment]
Welcome to GitHub @jack0194, When you're returning to GitHub after some time away, here's some recommended approach to get back into the swing of things:

1.Check your existing repositories for updates.
2.Review and update your GitHub profile.
3.Explore new projects or use templates.
4.Contribute to issues to get started.
5.Create your own projects if you have ideas. 
Ultimately, the best approach depends on your goals and interests.

**Reply by jack0194** [reply]
Thanks for your reply. So Let me just see if I have this right. The repositories is mainly for project updates, not project issues? (unless the update doesn't work) The issues section is where the problems get posted. So If I have this write start by contributing to known issues then move on from there.

**Reply by vishalkrishna8** [reply, marked as answer]
Yes, you've got it right!
1.Repositories:They store project code.
2.Issues:Report problems, suggest features, and discuss.

Start by fixing known issues to get involved, then consider larger contributions as you gain experience.
